internal/webhookhandler/helper.go

```diff
@@ -0,0 +1,27 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhookhandler
+
+import corev1 "k8s.io/api/core/v1"
+
+// Checks if Pod is already instrumented by checking Instrumentation InitContainer presence.
+func IsPodInstrumentationMissing(pod corev1.Pod) bool {
```

Comment:
I don't want to be nitpicking but a better name could be used here:  e.g. `IsAutoInstrumentationInjected`

Reply:
Agree, it's more meaningful ;)